clib/memprof_coq.std.ml

```diff
@@ -0,0 +1,144 @@
+let is_interrupted _ = false [@@inline]
+
+module Resource_bind = struct
+  let ( let& ) f scope = f ~scope
+end
+
+module Mutex_aux = struct
+  external unlock: Mutex.t -> unit = "caml_mutex_unlock"
+
+  (* Critical sections :
+     - Mutex.lock does not poll on leaving the blocking section
+       since 4.12.
+     - Never inline, to avoid theoretically-possible reorderings with
+       flambda.
+     - Inline the call to Mutex.unlock to avoid polling in bytecode.
```

Comment:
I guess by inline you mean redeclare the external

Reply:
This is @gadmm 's code, it was already in this form in `cThread`. I'll let him answer this question, but the code is verbatim from `memprof-limits`